pkg/testsuite/mock/wallet_transactions.go

```diff
@@ -426,37 +469,82 @@ func (w *Wallet) AllotManaFromInputs(transactionName string, allotments iotago.A
 		outputStates = append(outputStates, outputBuilder.MustBuild())
 	}
 
-	signedTransaction := lo.PanicOnErr(w.createSignedTransactionWithOptions(
+	signedTransaction := w.createSignedTransactionWithOptions(
 		transactionName,
 		WithAllotments(allotments),
 		WithInputs(inputStates),
 		WithOutputs(outputStates),
-	))
+	)
 
 	w.registerOutputs(transactionName, signedTransaction.Transaction)
 
 	return signedTransaction
 }
 
-func (w *Wallet) createSignedTransactionWithOptions(transactionName string, opts ...options.Option[builder.TransactionBuilder]) (*iotago.SignedTransaction, error) {
+func (w *Wallet) ClaimDelegatorRewards(transactionName string, inputName string) *iotago.SignedTransaction {
+	input := w.Output(inputName)
+	inputDelegation, ok := input.Output().(*iotago.DelegationOutput)
+	if !ok {
+		panic(fmt.Sprintf("output with alias %s is not *iotago.AccountOutput", inputName))
+	}
+
+	rewardMana, _, _, err := w.Node.Protocol.MainEngineInstance().SybilProtection.DelegatorReward(
+		inputDelegation.ValidatorAddress.AccountID(),
+		inputDelegation.DelegatedAmount,
+		inputDelegation.StartEpoch,
+		inputDelegation.EndEpoch,
```

Comment:
`EndEpoch` might be 0 when the Delegation Output is in its regular state, so this will not always have the intended behavior. We can fix that in another PR, though.